specs/core/1_shard-data-chains.md

```diff
@@ -47,14 +47,16 @@ This document describes the shard data layer and the shard fork choice rule in P
 | `BYTES_PER_SHARD_BLOCK_BODY` | `2**14` (= 16,384) |
 | `MAX_SHARD_ATTESTIONS` | `2**4` (= 16) |
 | `PHASE_1_GENESIS_EPOCH` | **TBD** |
-| `PHASE_1_GENESIS_SLOT` | get_epoch_start_slot(PHASE_1_GENESIS_EPOCH) |
+| `PHASE_1_GENESIS_SLOT` | **TBD** |
```

Comment:
Not quite relevant to this PR but i think "genisis" is a bad name here. Suggest "fork" -- `PHASE_1_FORK_*`